libdnf5-cli/output/repo_info.cpp

```diff
@@ -22,10 +22,25 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "key_value_table.hpp"
 #include "utils/string.hpp"
 
+#include "libdnf5-cli/tty.hpp"
```

Comment:
Can I ask why not to include directly `<iostream>`?

Reply:
Hmm, just a leftover, I've initially used something from that `tty` header, will change that...